scripts/create_addon

```diff
@@ -139,4 +220,75 @@ if [ "$PKG_IS_ADDON" = "yes" ] ; then
   else
     pack_addon
   fi
+}
+
+# need parameter
+if [ $# == 0 ]; then
+  usage 1
+fi
+
+# collect list of addons for building
+addons=()
+addons_drop=()
+show_only="false"
+export BUILD_INDENT=$((${BUILD_INDENT:-1}+$BUILD_INDENT_SIZE))
+
+# read addons from parameter list
+while [ $# -gt 0 ]; do
+  case $1 in
+    --help)      usage 0;;
+    --show-only) show_only="true";;
+    -*)          addons_drop+=" $(find_addons ${1:1})";;
+    *)           addons+=" $(find_addons $1)";;
+  esac
+  shift
+done
+
+# check environment and create toolchain
+$SCRIPTS/checkdeps
+setup_toolchain target 
+
+# build addons, by calling function build_addon with one addon, after another
+# (do not abort on build failure)
+addons_failed=""
+set +e
```

Comment:
Isn't this already set by sourcing `config/options` (specifically, `config/path`)?

Reply:
nope. `config/options` sets `set -e` (not +). i have to deactivate the return code check, because every failed addon, will abort the whole `create_addon`-script.